python/cudf/cudf/dataframe/string.py

```diff
@@ -795,6 +795,16 @@ def is_monotonic_decreasing(self):
             ).all()
         return self._is_monotonic_decreasing
 
+    def value_counts(self, method="sort"):
+        if method != "sort":
+            msg = "non sort based value_count() not implemented yet"
+            raise NotImplementedError(msg)
+        from cudf import Series
+
+        val_counts = Series(self, name=self.name)
+        res = val_counts.groupby(val_counts).count()
+        return res.index, res._column.astype("int64")
```

Comment:
Any reason we're typecasting to int64 here as opposed to leaving it as an int32?

Reply:
matching pandas behavior. pandas `value_counts` return a count series of type `int64`

Reply:
Yea understand, but given we've made the decision to limit our dataframe size to int32 rows anyway I think we should just return int32 here for now.

Reply:
@kkraus14 , removed the explicit typecast to int64.